README.md

```diff
@@ -129,7 +143,18 @@ Let's say the Posts in the previous question are available at `http://localhost:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all;
+    render json: posts;
+    if(@posts) {
+      console.log(@posts);
+    }
+    else {
+      console.log ('hi, im an error.');
+    }
+  end
+end
 ```
 
```

Comment:
wrong language! think back to the AJAX APIs lesson where we weirdly were writing js in a Rails app (possibly the source of your confusion here).

```js
.done( function(res){
  console.log(res)
})
.fail( function(res){
  console.log(res)
})
```